### DISCUSSION TITLE
WebUI won't start using fresh container in a VM?

### DISCUSSION THREAD

**Discussion by jtermi03** [sct/overseerr#948] [Help] [answered]
Hi, I'm new to docker so forgive any dumb questions here, I feel like this is a simple issue I just don't understand.

I have Docker running in a newly created Ubuntu VM on TrueNAS 12. I was able to install and access the WebUI for Portainer using [vm ip address]:9000, so I assume everything is ok so far.

The Overseerr image seems to deploy without errors using the command 

`sudo docker run -d -e LOG_LEVEL=info -e TZ=America/LosAngeles -p 5055:3000 -v /srv/config/overseerr:/app/config --restart unless-stopped sctx/overseerr:develop`

the log file shows "2021-02-16T13:44:51.832Z [info][Server]: Server ready on port 5055"

but when I navigate to [vm ip address]:5055, no connection can be established. Am I missing something?

**Comment by sct** [comment, marked as answer]
The internal port is 5055 not 3000. Your docker run command is wrong.

**Reply by jtermi03** [reply]
Yeah I noticed that shortly after posting this and tried it with 5055:5055, still no dice

EDIT: Just kidding, I had it as "5055:5050", that'll do it, thank you! Odd that the walkthrough I was using had 3000 as the internal port

**Reply by sct** [reply]
When we initially released alpha, it was internal port `3000` for about a week. I imagine the guide just hasn't updated since then. Happy to hear it's working now!

**Reply by jtermi03** [reply]
Ah, excellent, thanks for the quick reply! App looks great!
